tests/test_parser_utils.py

```diff
@@ -13,12 +14,242 @@
 import cwl_utils.parser.cwl_v1_1_utils
 import cwl_utils.parser.cwl_v1_2
 import cwl_utils.parser.cwl_v1_2_utils
+import cwl_utils.parser.utils
 from cwl_utils.errors import WorkflowException
 from cwl_utils.parser import load_document_by_uri
 
 from .util import get_data
 
 
+def test_static_checker_fail() -> None:
+    """Confirm that static type checker raises expected exception."""
+    with pytest.raises(ValidationException):
```

Comment:
Could these checks be made more specific about the expected exception message?

https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions

Reply:
Did it